packages/superset-ui-chart/test/components/MockChartPlugins.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ChartMetadata, ChartPlugin, ChartFormData } from '../../src';
+
+export const TestComponent = ({
+  message,
+  width,
+  height,
+}: {
+  message: string;
+  width: number;
+  height: number;
+}) => (
+  <div className="test-component">
+    <span className="message">{message || 'test-message'}</span>
+    <span className="dimension">{[width, height].join('x')}</span>
+  </div>
+);
+
+export const ChartKeys = {
+  DILIGENT: 'diligent-chart',
+  LAZY: 'lazy-chart',
+  SLOW: 'slow-chart',
+};
+
+export class DiligentChartPlugin extends ChartPlugin<ChartFormData> {
+  constructor() {
+    super({
+      metadata: new ChartMetadata({
+        name: 'diligent-chart',
```

Comment:
nit: Should we use the consts you defined above? Comment applies to the rest of the plugins.